reference/7.0/Microsoft.PowerShell.Core/About/about_Windows_PowerShell_Compatibility.md

```diff
@@ -38,58 +49,65 @@ Windows PowerShell Compatibility funtionality can be invoked in three ways:
+  "DisableImplicitWinCompat" = $false
+  "Microsoft.PowerShell:ExecutionPolicy" = "RemoteSigned"
+}
+$ConfigJSON | Out-File -Force $ConfigPath
+pwsh -settingsFile $ConfigPath
 ```
 
-In all three cases above a module can be blocked from loading using
-Windows PowerShell Compatibility by a `WindowsPowerShellCompatibilityModuleDenyList`
-setting in PowerShell configuration file. The default value of this setting:
-
-```json
-"WindowsPowerShellCompatibilityModuleDenyList":  ["PSScheduledJob","BestPractices","UpdateServices"]
-```
-
-When the first module is imported using Windows PowerShell Compatibility funtionality
-a `WinPSCompatSession` remoting session is created (using the same transport that
-PowerShell jobs use). This remoting session can be used for operations that
-do not work correctly on deserialized objects. The entire pipeline is executed
-in Windows PowerShell and only the final result is returned.
+When the first module is imported using Windows PowerShell Compatibility
+functionality a `WinPSCompatSession` remoting session is created using the same
+transport used for PowerShell jobs. This remoting session can be used for
+operations that do not work correctly on deserialized objects. The entire
+pipeline is executed in Windows PowerShell and only the final result is
+returned.
 
 ```powershell
 $s = Get-PSSession -Name WinPSCompatSession
-Invoke-Command -Session $s -ScriptBlock {"Running in Windows PowerShell version $($PSVersionTable.PSVersion)";<#Get-WinPSLiveObject | Set-WinPSLiveObject#>}
+Invoke-Command -Session $s -ScriptBlock {
+  "Running in Windows PowerShell version $($PSVersionTable.PSVersion)"
+}
 ```
 
 The background Windows PowerShell 5.1 process is created when the first module
-is imported using Windows PowerShell Compatibility funtionality and closed
-when the last such module is removed (using `Remove-Module`) or when
+is imported using Windows PowerShell Compatibility feature. The process is
+closed when the last such module is removed (using `Remove-Module`) or when
 PowerShell process exits.
 
-When a module is imported using Windows PowerShell Compatibility functionality,
-implicit remoting generates a proxy module in user `$env:Temp` directory
-and imports this proxy module into current PowerShell session.
-So the module's path (along with `(Get-Module -Name ServerManager).PrivateData.ImplicitRemoting`
-field) can be used to detect if a module was loaded using Windows PowerShell
-Compatibility funtionality.
+When a module is imported using the Windows PowerShell Compatibility feature,
+implicit remoting generates a proxy module in user `$env:Temp` directory and
+imports this proxy module into current PowerShell session. This allows
+PowerShell to detect that the module was loaded using Windows PowerShell
+Compatibility functionality.
+
+For more the latest information about module compatibility, see the
+[PowerShell 7 module compatibility](https://aka.ms/PSModuleCompat) list.
```

Comment:
This should likely be the full link instead of an aka.ms link since it's to docs and not likely to change frequently.

Reply:
Actually it may. As we add more lists of modules we may have to break this up  into its own TOC.